### PR TITLE
Use attributes on docs

### PR DESCRIPTION
**Pull request by jordisala1991**
Should be pedantic.

### CODE REVIEW

File: docs/reference/installation.rst [hunk 1]
```diff
@@ -125,63 +125,69 @@ And then create the corresponding entities, ``src/Entity/SonataMediaMedia``::
...
-        /**
-         * @ORM\Id
-         * @ORM\GeneratedValue
-         * @ORM\Column(type="integer")
-         */
-        protected $id;
+        #[ORM\Id]
+        #[ORM\Column(type: Types::INTEGER)]
+        #[ORM\GeneratedValue]
+        protected ?int $id = null;
```
Comment (phansys) [inline docs/reference/installation.rst:162]: IIMO, if we are forcing `id` to be `?int`, we should use the same declaration as return type in `getId()`.

File: docs/reference/installation.rst [hunk 2]
```diff
@@ -125,63 +125,69 @@ And then create the corresponding entities, ``src/Entity/SonataMediaMedia``::
...
+
+        public function getId()
```
Comment (jordisala1991) [inline docs/reference/installation.rst]:
```suggestion
        public function getId(): ?int
```

File: docs/reference/installation.rst [hunk 3]
```diff
@@ -125,63 +125,69 @@ And then create the corresponding entities, ``src/Entity/SonataMediaMedia``::
...
```
Comment (jordisala1991) [inline docs/reference/installation.rst]:
```suggestion
        public function getId(): ?int
```

File: docs/reference/installation.rst [hunk 4]
```diff
@@ -125,63 +125,69 @@ And then create the corresponding entities, ``src/Entity/SonataMediaMedia``::
...
```
Comment (jordisala1991) [inline docs/reference/installation.rst]:
```suggestion
        public function getId(): ?int
```

File: docs/reference/installation.rst (hunk 1 above)
Comment (jordisala1991) [reply docs/reference/installation.rst:162]: You’re right, applied.

**Review by VincentLanglet — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**